toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractProjectRunner.java

```diff
@@ -140,9 +158,17 @@ private void execute(final ITestPlan project, final TestRunner runner) {
 				}
 			}
 		}
+		if(shouldSendMail()){
+			sendEmailReport(project);
+		};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)